### PR TITLE
Update

### PR DESCRIPTION
**Pull request by Justman100**

### CODE REVIEW

**Comment by Justman100** [comment]
@saisatishkarra

**Review by itsikcircle — APPROVED** [review]

**Comment by itsikcircle** [comment]
You should also update `.github/workflows/build.yml` to test this version

**Comment by itsikcircle** [comment]
For the meantime, until this package is published, I've published a fork with the node engine changes:
https://www.npmjs.com/package/@itsikm/httpsnippet

**Comment by Justman100** [comment]
@saisatishkarra

**Comment by Justman100** [comment]
@saisatishkarra

File: package.json [hunk 1]
```diff
@@ -39,7 +39,7 @@
     "xmlhttprequest"
   ],
   "engines": {
-    "node": "^14.19.1 || ^16.14.2 || ^18.0.0 || ^20.0.0"
+    "node": "^14.0.0 || ^16.0.0 || ^18.0.0 || ^20.0.0 || ^21.0.0"
```
Comment (erunion) [inline package.json]: Honestly would recommend dropping Node 14 and 16 as they're both EOL. Should also probably start testing Node 22 too?

File: package-lock.json
Comment (erunion) [inline package-lock.json:1]: why'd this get deleted?

Comment (Justman100) [reply package-lock.json:1]: Because superfluous?

File: package.json (hunk 1 above)
Comment (Justman100) [reply package.json]: Yes, I'm already working on it

Comment (Justman100) [reply package.json]: DONE 🙂🥳

**Comment by Justman100** [comment]
@filfreire

**Comment by Justman100** [comment]
@filfreire

**Comment by filfreire** [comment]
Thank you for the PR, we will handle this in a separate one

**Comment by Justman100** [comment]
> Thank you for the PR, we will handle this in a separate one

Are you kidding me? Instead of just merging?

**Pull request closed without merging**